docs/src/pages/migrations.md

```diff
@@ -172,6 +172,11 @@ The following restructuring was done to achieve a more consistent grouping:
 
 - Webhook signatures now use the `rfc9421` protocol
 
+#### Rootless images
+
+From 3.0 onward, all Woodpecker images use a non-privileged user (`woodpecker`) by default.
```

Comment:
Does it make sense to put the version in here? Docs are versioned and the migration notes always reflect required changes of the next major anyway?